GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -204,6 +205,46 @@ void Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat,
     }
 }
 
+// This gives us a really coarse bounding sphere given the already calcuated bounding box.  This won't be a tight-fitting sphere because it is based on the bounding box.  We can revisit this later if we decide we need a tighter sphere.
+const float *Mesh::getBoundingSphereInfo() {
+    if (have_bounding_box_ == false) {
+        getBoundingBoxInfo();
+    }
+
+    if (have_bounding_sphere_ == true) {
+        return bounding_sphere_info_;
+    }
+
```

Comment:
Very minor here. I think if(!have_bounding_box) and if(have_bounding_sphere_) would be a little better?